### ISSUE TITLE
Serial commands are completely undiscoverable

### ISSUE THREAD

**Issue by xoseperez** [xoseperez/espurna#338]
Originally reported by: **Fake Name (Bitbucket: [fake-name](https://bitbucket.org/fake-name), GitHub: [fake-name](https://github.com/fake-name))**

----------------------------------------

So I just flashed Espurna to a Sonoff Pow, and I'd like to set it up using the serial interface.

However, trying to do so, **I have no idea what any of the serial commands are**.

I've tried googling "espurna serial interface" and all the variations of that I can think of, without any result. In addition, the serial CLI itself seems to have no facility for help. I've tried `help`, `?`, and every single single-letter in the alphabet as a command, and they *all* return `-ERROR unknown command`.

Searching the source for that exact string points me to one of the files in Embedis (`Embedis.cpp`), where none of the commands are actually defined.

So, basically, how the devil does the serial command interface work? It seems to me, at minimum, it should probably support at least `help` or `?`, or some sort of way for figuring out what it does.
